src/Parser/Core/Modules/EnemyInstances.js

```diff
@@ -5,11 +5,11 @@ const debug = true;
 
 class EnemyInstances extends Enemies {
   getEntity(event) {
-    if (event.targetIsFriendly) {
+    if (event.targetIsFriendly && event.sourceIsFriendly) {
```

Comment:
Why did you change this? I don't think this is supposed to include damage to friendly targets (e.g. other players or player pets), but your change would lead to including friendly targets given the source isn't friendly (e.g. another player hit by the boss).